app/views/includes/search-form.nunjucks

```diff
@@ -10,19 +10,13 @@
       {% if errorMessage %}
         <h2>{{ searchErrorLabel }}</h2>
         <p>{{ errorMessage }}</p>
-      {% elif gps.length === 0 %}
-         <h2>We can't find a surgery using '{{search}}'</h2>
-         <p>Check the text you entered is right. You get better results if you enter a full name. Or you can search with a postcode instead.</p>
       {% else %}
-        <h2>Find your GP surgery</h2>
-        <p>You need to do this to go to your booking system. Enter the name of your surgery, the name of your GP or a postcode.</p>
+        {{ noResultsMessage | safe }}
```

Comment:
I felt OK using it in the previous work as there was no user input in the string being piped through safe. This is not the case here. I'm not sure if this introduces a vulnerability here. A quick check suggests not but I'd want to be sure.